build-utils/protocol-base-mspec/src/main/java/org/apache/plc4x/plugins/codegenerator/language/mspec/model/definitions/DefaultComplexTypeDefinition.java

```diff
@@ -65,6 +65,11 @@ public boolean isAbstract() {
             .collect(Collectors.toList());
     }
 
+    public List<AbstractField> getAbstractFields() {
```

Comment:
we should add this method here to `org.apache.plc4x.plugins.codegenerator.types.definitions.ComplexTypeDefinition` and add a `Override` annotation
```suggestion
    @Override
    public List<AbstractField> getAbstractFields() {
```